src/controller/python/chip/interaction_model/__init__.py

```diff
@@ -81,6 +81,8 @@ class Status(enum.IntEnum):
     PathsExhausted = 0xc8
     TimedRequestMismatch = 0xc9
     FailsafeRequired = 0xca
+    InvalidInState = 0xcb
+    NoCommandResponse = 0xcc
```

Comment:
Could you add it to https://github.com/project-chip/connectedhomeip/blob/master/src/lib/format/protocol_messages.matter#L224 as well?